### PR TITLE
Gaj/feature/header

### PR DESCRIPTION
**Pull request by kelsiebesingeryeh**
#### Purpose:
 > Check all that apply.
- [X] Feature
- [ ] Refactor
- [ ] Bug Fix 
- [ ] Clean Up

### What does this PR do?
 > Clearly and concisely describe changes made. 
Add a header component along with a hamburger menu component to display the dropdown of items a user can select.

Issues closed #

> Describe your code changes in detail for reviewers.
Added a npm package for the hamburger menu, then added state w/hooks to conditionally render when the hamburger menu is toggled, it should show the dropdown items.

Header component => look at the JS and scss files

npm start in the browser window

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -1,11 +1,34 @@
...
+      />
+      <Route
+      exact
+      path="/login"
+      render={() => <Login/>}
+      />
+      <Route
+      exact
+      path='/user-landing'
+      render={() => <UserLanding/>}
```
Comment (JeffKersting) [inline src/components/App/App.js:30]: Going forward, will we need App useState hook to track the user id (to pass to landing page)? Possibly pass the hook through to login page?

File: src/components/Header/Header.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import { Link } from 'react-router-dom';
+import Hamburger from "hamburger-react";
+import './_Header.scss';
+
+const Header = () => {
+  const [isOpen, setOpen] = useState(false)
+  
+  return (
+    <header className='mobileHeader'>
+      <Hamburger toggled={isOpen} toggle={setOpen} />
```
Comment (JeffKersting) [inline src/components/Header/Header.js:11]: This looks great!

**Review by JeffKersting — APPROVED** [review]

File: src/components/App/App.js (hunk 1 above)
Comment (gaj23) [reply src/components/App/App.js:30]: I don't think so, personally. I'd rather that ownership be limited to as many places as possible. Additionally, there might be some other hooks that could be beneficial. I'm thinking useContext.

File: src/components/App/App.scss [hunk 3]
```diff
@@ -1 +0,0 @@
-
```
Comment (gaj23) [inline src/components/App/App.scss:1]: This file should probably be deleted.

File: src/components/Header/Header.js (hunk 2 above)
Comment (gaj23) [reply src/components/Header/Header.js:11]: Oh my gosh, this npm package is so worth it.

File: src/index.scss [hunk 4]
```diff
@@ -1,3 +1,12 @@
 @import './variables';
 @import './components/App/App';
-@import './components/User-Landing-Page/landing'
+@import './components/Header/Header';
```
Comment (gaj23) [inline src/index.scss:3]: Do the underscores not need to be included before all of these files?

**Review by gaj23 — APPROVED** [review]
I'm excited to get this merged!

**Pull request merged**